samples/matter/light_bulb/README.rst

```diff
@@ -339,6 +339,61 @@ Upgrading the device firmware
 
 To upgrade the device firmware, complete the steps listed for the selected method in the :doc:`matter:nrfconnect_examples_software_update` tutorial in the Matter documentation.
 
+AWS IoT Integration
```

Comment:
I'm missing here the information that Wi-Fi is the only supported networking technology and nRF7002 DK is the only supported platform. I know we have the example build command, but generally the sample can work with Thread as well, so it would be good to clearly mention that the support if for Wi-Fi only.

Reply:
Added information on start of the sample docs.